django/db/models/expressions.py

```diff
@@ -1174,6 +1174,9 @@ def as_sql(self, compiler, connection):
             if hasattr(output_field, "get_placeholder"):
                 return output_field.get_placeholder(val, compiler, connection), [val]
         if val is None:
+            if output_field.__class__.__name__ == "JSONField":
+                # Return JSON null in case the field is JSONField.
+                return "'null'", []
```

Comment:
I'm not convinced this is the best way to do this and there appears to be a growing consensus that we should pursue introducing a `JSONNull` (see https://code.djangoproject.com/ticket/35381#comment:5, https://code.djangoproject.com/ticket/35381#comment:10, https://code.djangoproject.com/ticket/36445#comment:11)

Reply:
@GoyalRocks007 Sarah's given the complete context 👍 If you need assistance jump on the Discord (chat). Sarah is busy managing all the other PRs so won't be able to help you much but there may be others on Discord who will be able to help.